### PR TITLE
Add filtering example to cloud video calling

### PR DESCRIPTION
**Pull request by GiovanniBeriSinch**
RTC-6825

### CODE REVIEW

**Comment by GiovanniBeriSinch** [comment]
I changed the example from the one in https://git.sinch.se/client-sdk/rtc_sdk_ios/-/blob/v4.2/Samples/SinchVideoFilter/VideoFilter.m#L8, because I did not manage to make remote callback work by rendering the filtered image to a new pixel buffer. While the exact same code worked properly for the local video callback, I kept getting a red screen by applying the callback to the remote video callback (see image below). I tried to change the retain-release logic with no success, I think it's just easier to do it this way.


IMG_F7B5AD7B4260-1

File: docs/voice/voice-ios-cloud/voice-ios-cloud-video-calling.md [hunk 1]
```diff
@@ -131,21 +131,46 @@ The Sinch SDK provides access to the raw video frames of the remote and local vi
...
-  
-  // Use GCD dispatch to process frame async.
-  dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^{
-    CVPixelBufferRef outputPixelBuffer = ... // Perform actual processing.
-    completionHandler(outputPixelBuffer);
-    CVPixelBufferRelease(outputPixelBuffer); // Release your intermediate processing output buffer.
-    CVPixelBufferRelease(inputPixelBuffer); // Release the original input frame buffer.
-  });
+- (void)onFrame:(CVPixelBufferRef)pixelBuffer completionHandler:(void (^)(CVPixelBufferRef))completionHandler {
+  if (pixelBuffer) {
```
Comment (dmipon) [inline docs/voice/voice-ios-cloud/voice-ios-cloud-video-calling.md:139]: replace with `if (!pixelBuffer) {` it reduces braces nesting level

Comment (GiovanniBeriSinch) [reply docs/voice/voice-ios-cloud/voice-ios-cloud-video-calling.md]: done

**Comment by dmipon** [comment]
LGTM

**Comment by dmipon** [comment]
LGTM

**Pull request merged**